script/provider_migrate_to_hostname_operator.md

```diff
@@ -82,5 +83,5 @@ The last line specifies the image tag and should correspond to whatever version
 To install the operator into kubernetes perform the following from the `/_docs` directory.
 
 ```
-kubectl kustomize ./kustomize/akash-hostname-operator | kubectl apply -f -
+kubectl kustomize _docs/kustomize/akash-hostname-operator | kubectl apply -f -
```

Comment:
If you look up on the lines above states you are in the `/_docs` directory. So if we are going to make this change then we need to remove that line.